src/App.tsx

```diff
@@ -1,239 +1,48 @@
-              <Divider className="pf-u-mt-lg pf-u-mb-lg" />
-              <CatalogSection
-                sectionName="quick-starts"
-                toggleFavorite={toggleFavorite}
-                sectionCount={quickStarts.length}
-                sectionTitle="Quick starts"
-                sectionDescription="Step-by-step instructions and tasks"
-                sectionQuickStarts={quickStarts}
-                activeQuickStartID={activeQuickStartID}
-                allQuickStartStates={allQuickStartStates}
-              />
-              <Divider className="pf-u-mt-lg pf-u-mb-lg" />
-              <CatalogSection
-                sectionName="learning-paths"
-                toggleFavorite={toggleFavorite}
-                sectionCount={learningPaths.length}
-                sectionTitle="Learning paths"
-                sectionDescription="Collections of learning materials contributing to a common use case"
-                sectionQuickStarts={learningPaths}
-                activeQuickStartID={activeQuickStartID}
-                allQuickStartStates={allQuickStartStates}
-              />
-              <Divider className="pf-u-mt-lg pf-u-mb-lg" />
-              <CatalogSection
-                sectionName="other-content-types"
-                toggleFavorite={toggleFavorite}
-                sectionCount={other.length}
-                sectionTitle="Other content types"
-                sectionDescription="Tutorials, videos, e-books, and more to help you build your skills"
-                sectionQuickStarts={other}
-                activeQuickStartID={activeQuickStartID}
-                allQuickStartStates={allQuickStartStates}
-              />
-            </SidebarContent>
-            <SidebarPanel
-              variant="sticky"
-              className="pf-u-background-color-200 pf-u-pl-lg pf-u-pl-0-on-lg"
-            >
-              <TableOfContents
-                defaultActive="bookmarks"
-                linkItems={[
-                  {
-                    id: 'bookmarks',
-                    label: `Bookmarks (${bookmarks.length})`,
-                  },
-                  {
-                    id: 'documentation',
-                    label: `Documentation (${documentation.length})`,
-                  },
-                  {
-                    id: 'quick-starts',
-                    label: `Quick starts (${quickStarts.length})`,
-                  },
-                  {
-                    id: 'learning-paths',
-                    label: `Learning paths (${learningPaths.length})`,
-                  },
-                  {
-                    id: 'other-content-types',
-                    label: `Other content types (${other.length})`,
-                  },
-                ]}
-              />
-            </SidebarPanel>
-          </Sidebar>
-        </div>
-      </PageSection>
-    </PageGroup>
+    <AppContext.Provider
+      value={{
+        onNavigate: creatorEnabled
+          ? (newPart) => {
```

Comment:
How would I go about doing that? Just updating `fec.config.js`?

Reply:
Yeah, if you look at the current fec.config.js, it already exposes some extra components. The creator should be the same thing. It should work as a standalone module. independent from the viewer. It can share components but it should be its "own app".